source/options.rst

```diff
@@ -103,58 +100,62 @@ Now you can create the cluster following the :ref:`regular installation instruct
 Modifying options for the existing cluster
 ------------------------------------------
 
-For example, you can change ``max_connections`` option in a ``postgresql.conf``
-configuration file with the following ``postgres-ha.yaml`` contents:
-
-.. code:: yaml
-
-   ---
-   dcs:
-     postgresql:
-       parameters:
-         max_connections: 50
+If you need to update cluster’s configuration settings, you should modify
+settings in the ``<clusterName>-pgha-config`` ConfigMap.
 
-..note:: ``dsc.postgresql`` subsection means that option will be applied globally to
-         ``postgresql.conf`` of all database servers.
+.. note:: This ConfigMap contains ``<clusterName>-dcs-config`` configuration
+   applied globally to ``postgresql.conf`` of all database servers, and
+   local configurations for the PostgreSQL cluster database servers:
+   ``<clusterName>-local-config`` for the current primary,
+   ``<clusterName>-repl1-local-config``for the first replica, and so on.
 
-You can create a ConfigMap from this file. The syntax for ``kubectl create configmap`` command is:
+For example, let's change the ``max_connections`` option in a globally applied
+``postgresql.conf`` configuration file for the cluster named ``cluster1``. 
+Edit the ``cluster1-pgha-config`` ConfigMap with the following command:
 
 .. code:: bash
 
-   kubectl -n <namespace> create configmap <configmap-name> --from-file=postgres-ha.yaml
-
-ConfigMap name should include your cluster name and a dash as a prefix
-(``cluster1-`` by default). 
+   $ kubectl edit -n pgo configmap cluster1-pgha-config
 
-The following example defines ``cluster1-custom-config`` as the ConfigMap name:
-
-.. code:: bash
+This will open the ConfigMap in a local text editor of your choice. Make sure
+to modify it as follows:
 
-   $ kubectl create -n pgo configmap cluster1-custom-config --from-file=postgres-ha.yaml
+.. code:: yaml
 
-To view the created ConfigMap, use the following command:
+   ...
+   cluster1-dcs-config: |
+     postgresql:
+       parameters:
+        ...
+        max_connections: 50
+        ...
 
-.. code:: bash
+Now :ref:`restart the cluster<operator-pause>` to ensure the update took effect.
```

Comment:
operator-pause?

Reply:
That's a link to source/pause.rst (the title on this page says "Pause/resume PostgreSQL Cluster")